### PR TITLE
Missing attributes

### PR DESCRIPTION
**Pull request by ramarag**
@wesh @stephentoub  @joperezr  PTAL

### CODE REVIEW

File: src/System.Runtime/ref/System.Runtime.cs [hunk 1]
```diff
@@ -344,6 +344,31 @@ public abstract partial class Attribute
         public virtual bool IsDefaultAttribute() { return default(bool); }
         public virtual bool Match(object obj) { return default(bool); }
         public virtual object TypeId { get { return default(bool); } }
+        public virtual object TypeId { get { return default(object); } }
```
Comment (AlexGhiondea) [inline src/System.Runtime/ref/System.Runtime.cs]: We should keep a single entry for TypeId -- and it should be the one that returns a default(object) (i.e. the one you added).
I think I added the one that mistakenly returns default(bool).

I am surprised that you did not get a build error for the duplicate property.

File: src/System.Runtime/src/ApiCompatBaseline.uap101aot.txt [hunk 2]
```diff
@@ -431,4 +431,16 @@ MembersMustExist : Member 'System.Threading.Tasks.Task.Dispose(System.Boolean)'
 CannotRemoveBaseTypeOrInterface : Type 'System.Threading.Tasks.Task<TResult>' does not implement interface 'System.IDisposable' in the implementation but it does in the contract.
 MembersMustExist : Member 'System.Attribute.IsDefaultAttribute()' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.Attribute.Match(System.Object)' does not exist in the implementation but it does exist in the contract.
-MembersMustExist : Member 'System.Attribute.TypeId.get()' does not exist in the implementation but it does exist in the contract.
\ No newline at end of file
+MembersMustExist : Member 'System.Attribute.TypeId.get()' does not exist in the implementation but it does exist in the contract.
```
Comment (AlexGhiondea) [inline src/System.Runtime/src/ApiCompatBaseline.uap101aot.txt]: I think that once we remove one of the properties for typeId, there should only be one of them here.

File: src/System.Runtime/tests/System.Runtime.Tests.csproj [hunk 3]
```diff
@@ -166,6 +166,10 @@
     <Compile Include="$(CommonTestPath)\System\Runtime\Serialization\Formatters\BinaryFormatterHelpers.cs">
       <Link>System\Runtime\Serialization\Formatters\BinaryFormatterHelpers.cs</Link>
     </Compile>
+    <Compile Include="System\Runtime\CompilerServices\RuntimeHelpersTests1.7.cs" />
```
Comment (AlexGhiondea) [inline src/System.Runtime/tests/System.Runtime.Tests.csproj]: I believe the convention is to add .netstandard1.7 before the extension.

File: src/System.Runtime/ref/System.Runtime.cs (hunk 1 above)
Comment (ramarag) [inline src/System.Runtime/ref/System.Runtime.cs]: oops my bad did not pay close attention when resolving the merge conflict, will fix it

File: src/System.Runtime/tests/System/Runtime/Tests1.7.cs [hunk 4]
```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
```
Comment (joperezr) [inline src/System.Runtime/tests/System/Runtime/Tests1.7.cs]: Should we change the name of the file here to be something like: RuntimeHelpersTests.netstandard1.7.cs?

Comment (joperezr) [inline src/System.Runtime/tests/System/Runtime/Tests1.7.cs]: In fact, looks like you are already adding that file in the CompilerServices folder, should we just move these tests over there?

---
In reply to: [81802069](https://github.com/dotnet/corefx/pull/12337#discussion_r81802069) [](ancestors = 81802069)

**Comment by joperezr** [comment]
Test Innerloop Ubuntu14.04 Debug Build and Test please (Infrastructure issue: Build timed out)

**Comment by ramarag** [comment]
@dotnet-bot test this

**Comment by ramarag** [comment]
@dotnet-bot test this please

**Comment by ramarag** [comment]
@dotnet-bot test Innerloop CentOS7.1 Debug Build and Test please

File: src/System.Runtime/tests/System/Runtime/Tests1.7.cs (hunk 4 above)
Comment (ramarag) [inline src/System.Runtime/tests/System/Runtime/Tests1.7.cs]: removed the file for now

**Pull request merged**